pkg/sentry/config/config_test.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"testing"
+
+	dapr_config "github.com/dapr/dapr/pkg/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConfig(t *testing.T) {
+	t.Run("valid FromConfig, empty config name", func(t *testing.T) {
+		defaultConfig := getDefaultConfig()
+		c, _ := FromConfigName("")
+
+		assert.Equal(t, defaultConfig, c)
+	})
+
+	t.Run("valid default config, self hosted", func(t *testing.T) {
+		defaultConfig := getDefaultConfig()
+		c, _ := getSelfhostedConfig("")
+
+		assert.Equal(t, defaultConfig, c)
+	})
+
+	t.Run("parse configuration", func(t *testing.T) {
+		daprConfig := dapr_config.Configuration{
+			Spec: dapr_config.ConfigurationSpec{
+				MTLSSpec: dapr_config.MTLSSpec{
+					Enabled:          true,
+					WorkloadCertTTL:  "5s",
+					AllowedClockSkew: "1h",
```

Comment:
nit: looks like two numbers are switched :) even if the test result will not be changed.

Reply:
Ah, its just values for the sake of testing duration, doesn't supposed to be real..